src/duckstation-qt/gamelistwidget.cpp

```diff
@@ -47,13 +47,27 @@ static constexpr int VIEW_MODE_LIST = 0;
 static constexpr int VIEW_MODE_GRID = 1;
 static constexpr int VIEW_MODE_NO_GAMES = 2;
 
-static constexpr int ICON_SIZE_STEP = 4;
-static constexpr int MIN_ICON_SIZE = 16;
-static constexpr int MAX_ICON_SIZE = 80;
+static constexpr int GAME_ICON_MIN_SIZE = 16;
+static constexpr int GAME_ICON_DEFAULT_SIZE = 16;
+static constexpr int GAME_ICON_MAX_SIZE = 80;
+static constexpr int GAME_ICON_SIZE_STEP = 4;
+// Nasty magic number here, +8 gets us a height of 24 at 16 icon size, which looks good.
+static constexpr int GAME_ICON_EXTRA_SIZE = 8;
+static constexpr int GAME_ICON_PADDING = 3;
```

Comment:
This looks fine on macOS and matches the old row spacing (before the icon scaling refactor, which made the rows a bit taller), but I have no idea if it looks decent on other platforms too. Setting it to 4 looks ok too if you prefer a roomier layout.